content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Add custom methods to your models'
+metaTitle: 'Prisma Client extensions: model (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom methods to your models.
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `model` extension type to add methods to a specific model in your schema or to all models in your schema.
+
+Possible uses for `model` include the following:
+
+- New operations to operate alongside existing Prisma Client operations, such as `findMany`
+- Encapsulated business logic
+- Repetitive operations
+- Model-specific utilities
+
+</TopBlock>
+
+## Enable the preview feature
+
+Before you create Prisma Client extensions, you must enable the `clientExtensions` feature flag in the `generator` block of your `schema.prisma` file, as follows:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["clientExtensions"]
+}
+```
+
+## Add a custom method to a specific model
+
+To extend a specific model in your schema, use the following structure. This example adds a method to the `user` model.
+
+```ts
+const xprisma = prisma.$extends({
+  name?: '<name>',
+  model?: {
+    user: { ... }   // in this case, we extend the `user` model
+  },
+});
+```
+
+- `name`: an optional field that you can use to name the extension. This name appears in error logs.
```

Comment:
I'd find it useful to have a sentence introducing this bullet point list, I was a bit confused about what it was

Reply:
Good point!